### PR TITLE
Update cluster-api-provider-ibmcloud meeting zoom link, time and notes document link

### PR DESCRIPTION
**Pull request by Karthik-K-N**
**Which issue(s) this PR fixes**:

This PR updates few things related to cluster-api-provider-ibmcloud
1. Update meeting timing form Friday 9:00 to Tuesday 10:00
2. Update meeting zoom link
3. Update the meeting notes link with redirect url , Ref [PR](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1986)

Fixes #

### CODE REVIEW

**Review by mkumatag — APPROVED** [review]
/lgtm

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by Karthik-K-N** [comment]
/assign @BenTheElder 
Could you please help.

File: sig-cluster-lifecycle/README.md [hunk 1]
```diff
@@ -154,8 +154,8 @@ Cluster API infrastructure provider for IBM Cloud
   - Slack: [#cluster-api-ibmcloud](https://kubernetes.slack.com/messages/cluster-api-ibmcloud)
   - [Mailing List](https://groups.google.com/a/kubernetes.io/g/sig-cluster-lifecycle)
 - **Meetings:**
-  - Cluster API Provider IBM Cloud office hours: [Fridays at 09:00 IST](https://zoom.us/j/9392903494) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09%3A00&tz=IST).
-    - [Meeting notes and Agenda](https://docs.google.com/document/d/1oWnqXy1VFv0E3kovQoZfS6IlVP0L4eaQsN-2HYC_6_A).
+  - Cluster API Provider IBM Cloud office hours: [Tuesdays at 10:00 IST](https://zoom.us/j/508079177) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10%3A00&tz=IST).
```
Comment (BenTheElder) [inline sig-cluster-lifecycle/README.md:157]: I assume this is still a meeting in the community account(s)?

**Review by BenTheElder — COMMENTED** [review]
/approve
/hold

File: sig-cluster-lifecycle/README.md (hunk 1 above)
Comment (mkumatag) [reply sig-cluster-lifecycle/README.md:157]: yes, it is shared with SIG cloud provider, courtesy @elmiko

**Pull request merged**